spec/requests/user/user_settings_spec.rb

```diff
@@ -88,6 +88,11 @@
         expect(response.body).to include(*titles)
       end
 
+      it "includes contact us on RSS page properly" do
+        get user_settings_path(:extensions)
+        expect(response.body).to include('<a href="/contact">')
```

Comment:
I think it'd be great to assert on the whole translation:

```suggestion
        expect(response.body).to include(I18n.t("contact_prompts.if_any_questions_html"))
```

This is actually one advantage of using I18n string, we don't need to assert on (parts of) the copy directly.